### ISSUE TITLE
Shift+Click - jira link - can't login to page

### ISSUE THREAD

**Issue by nmat** [wavebox/waveboxapp#1001]
* Wavebox Version: 4.9.1
* Operating System & Version: Windows 10
* Account type (if applicable): MS Teams and Jira

**Expected & actual behavior**
Open link in external window and login to check link. (would be awesome if it knew the login from other tabs if used also.

Gives error and can't login for some reason. Nothing happens.

**Steps to reproduce**
Keep a login for jira on one service tab
Open MS teams and get a link for a issue
Shift+Click the link to open in "new wavebox window"
Error comes up and when trying to login nothing happens.
Refreshing the page wants to leave or stay on page.

**Is the bug persistent or intermittent?**
Persistent

**Additional information**
Only tested on these 2 services. Might be other places where this happens. Tested to open in default browser and that works.

I can fill in the forms on page and click on login. But nothing happens after that. Does not show if it is thinking when trying to login or anything. For me this would make it so much easier since I do not have to change the active service and be more dynamic when doing multitasking inside wavebox.

Tested to right click -> Open link with wavebox -> New window

Logged in and now it works with shift+click the link. However I wonder why it did not work to login on first try for Shift+Click?

**Comment by Thomas101** [comment]
It could be that you're crossing a sandbox and this is causing some kind of issue :-/

Are the two services in under the same account or under different ones? If they are under the same service, does either of them have the `Sandbox Service` flag enabled under advanced?

**Comment by nmat** [comment]
These are 2 different services and not sandboxed.

Thing is.....right click -> open in wavebox -> new window (works)

Shift+Click - Opens page but does not allow me to login. I do not see that it tries to login. I can fill in information but nothing happens on try to login. My guess is that the window will try to use information from current service to open the link....including credentials

**Comment by Thomas101** [comment]
Not the problem I was thinking about then! Just so I can make sure I'm using the same as you, 

* Under `Settings` > `Links` what's your Shift+Click behaviour?
* Under `Account Settings` > `Links` are there any window open rules setup and if so what are they?

**Comment by nmat** [comment]
Under settings -> links I have for shift+click - default behaviour

Under account settings -> links I have 0 window open rules setup

and ask each time in there also

**Comment by Thomas101** [comment]
Struggling to reproduce this one.

* I've got the two in the same account...
![Screenshot 2019-04-05 at 09 08 28](https://user-images.githubusercontent.com/103586/55613380-7b72d900-5782-11e9-9860-8338827fae46.png)
* Then Shift+Clicked on an issue link in Teams `https://tomtest1084.atlassian.net/browse/DEMO-7` in my instance
* I get to the issue in the popup window...
![Screenshot 2019-04-05 at 09 10 17](https://user-images.githubusercontent.com/103586/55613461-a9581d80-5782-11e9-9396-1277152f108b.png)


If you try it again does it still happen? Maybe try adding the accounts a second time and giving it a try with that instance, just in case it's some cached data :-/

**Comment by nmat** [comment]
I am using it a bit different. Now when I tested it worked just fine not sure if it is because I told chrome to remember me on this computer for the login or what.

I will close this off right now and reopen if it occurs again.

**Comment by nmat** [comment]
Ok, so.....I managed to narrow it down. And also found the problem.

When I click on Shift+Click and hit login. There is no bar or indicator that it is trying to login. So therefore from my eyes it looks like the window is doing nothing but it does load in background.

For example if you open link with open in wavebox -> new window

then a bar comes up showing the progress of login. But it does not do that on shift+click.

If I just waited a while letting it run in the background then the login works. So in this case the action works but my eyes was cheating me thinking nothing was happening because there is no indicator of trying to do the login.
